services/codeclimate/codeclimate-coverage.service.js

```diff
@@ -3,14 +3,14 @@
 const Joi = require('joi')
 const { BaseJsonService, NotFound } = require('..')
 const { coveragePercentage, letterScore } = require('../color-formatters')
-const { keywords, fetchRepo } = require('./codeclimate-common')
+const { keywords, isLetterGrade, fetchRepo } = require('./codeclimate-common')
```

Comment:
Just to note a potential follow up that it might even be worth elevating this to broader validators at some point, as we do (or at least soon will) have some other grade data (Sonar comes to mind again)

Reply:
When we do this, one thought: some of these have "E"'s in them, and some don't.